packages/storybook/src/stories/ino-table/ino-table.stories.ts

```diff
@@ -23,113 +24,123 @@ export default {
       return () => tables.forEach(t => t.removeEventListener('sortChange', sortChangeHandler));
     })
     return story();
-  }]
+  }],
+  argTypes: {
+    sortColumnId: {
+      options: ['id', 'name', 'release', 'box-office', 'rating'],
+      control: {
+        type: 'select'
+      },
+    },
+  },
+  args: {
+    loading: false,
+    sortColumnId: 'id',
+    noHover: false,
+    stickyHeader: false,
+    sortDirection: 'asc',
+  },
 };
 
-export const Playground = args => html`
+const tableContent = html`
+    <tr slot="header-row"><ino-table-header-cell column-id="id" sort-start="asc"  label="ID"></ino-table-header-cell>
+      <ino-table-header-cell column-id="name" sort-start="asc" label="Name"></ino-table-header-cell>
+      <ino-table-header-cell column-id="release" label="Release Date"></ino-table-header-cell>
+      <ino-table-header-cell column-id="box-office" label="Box Office"></ino-table-header-cell>
+      <ino-table-header-cell column-id="rating" label="Rating" not-sortable></ino-table-header-cell>
+      <ino-table-header-cell column-id="another1" label="Another Column" not-sortable></ino-table-header-cell>
+      <ino-table-header-cell column-id="another2" label="Another Column" not-sortable></ino-table-header-cell>
+    </tr>
```

Comment:
Can you take a look on that issue?
![image](https://user-images.githubusercontent.com/23154336/173514991-dd120722-3202-46e9-8ce9-64ef1b70ad70.png)